src/applications/mhv-secure-messaging/tests/e2e/pages/PatientComposePage.js

```diff
@@ -14,6 +15,8 @@ class PatientComposePage {
 
   sendMessage = (mockRequest, mockResponse = mockDraftMessage) => {
     cy.intercept('POST', Paths.SM_API_EXTENDED, mockResponse).as('message');
+    // Note that we don't need specific event names in the response
+    cy.intercept('POST', Paths.UUM_API_BASE, mockUumResponse).as('uum');
```

Comment:
Note: We don't test UUMs in cypress, but this removes the error when there is no response from the API.